README.md

```diff
@@ -0,0 +1,29 @@
+# LiligoS3Dash
+
+## Init Git Submodules
+
+We use submodules to manage components. To download the deps, follow the instructions below:
+
+```
+git submodules init
+git submodules update
+```
+
+## Environment setup
+
+Follow [ESP32 installation guide](https://docs.espressif.com/projects/esp-idf/en/v4.3.1/esp32/get-started/index.html#installation-step-by-step) to setup your local environment. 
+
+## Enable BT 
+
+```
+idf.py menuconfig
+```
+
+In order for this to launch, you may need to follow the on-screen instructions to install `windows-curses`. 
+
+In menufconfig, select `Component config -> Bluetooth` and enable Bluetooth (Bluedroid). Save and exit.
```

Comment:
I believe the missing feature causing your build error is that BLE 4.2 feature is not enabled. It is under menuconfig as well. May worth mentioning. In fact with latest `sdkconfig.default` patch, these should automatically enabled now.

Reply:
Now that this is included in `sdkconfig.defaults`, I think we just need to instruct people to run `idf.py menuconfig` and immediately exit right?

Reply:
I think so.
I suspect even idf.py menuconfig step can be omit, but I don't have a machine to test it from start to finish now. There is no harm to run menuconfig anyway so it is OK we keep it.

Reply:
Confirmed it works without explicitly running menuconfig.